addons/mail/static/src/discuss/core/common/thread_model_patch.js

```diff
@@ -7,13 +8,39 @@ import { registry } from "@web/core/registry";
 
 const commandRegistry = registry.category("discuss.channel_commands");
 
+patch(Thread, {
+    sortOnlineMembers(m1, m2) {
+        return m1.persona.name?.localeCompare(m2.persona.name) || m1.id - m2.id;
+    },
+});
```

Comment:
Due to typedoc limitation, we cannot add static method in interfaces.

One solution in Discuss code of models is to avoid static methods as much as possible, especially in overrides.
To do so, we can move `sortOnlineMembers()` to an instance method of `Store`.

Reply:
I move `sortOnlineMembers` in store patch, so that this is an instance method and it has proper typing.

Solely due to typing issues, adding new `static` methods should be avoided as much as possible in business code. We should add instance methods and field attributes almost exclusively.